### PR TITLE
Simple loading screen

### LINKED ISSUE #32: Add loading/processing page/indicator to the frontend
Add some kind of 'processing' indicator to show the API is retrieving a result before the Results page is displayed.

### PR DESCRIPTION
**Pull request by jsjohnstone**
This is a simple loading screen: https://share.getcloudapp.com/p9uwQwR5

It's in a separate component, so it's very easy for us to style/improve what it looks like, but the functinoality is done 👍

### CODE REVIEW

**Comment by abirdonwire** [comment]
Looks great, and just what we need! 
Do you need any help with the checks?

File: src/components/URLInput/URLInput.js [hunk 1]
```diff
@@ -58,8 +61,10 @@ function URLInput(props) {
           return Promise.reject(error);
         }
         props.setRequest(data);
+        props.setLoading(false);
```
Comment (abirdonwire) [inline src/components/URLInput/URLInput.js:64]: All looks great.
I can try and help on the failing checks if you like?

**Review by abirdonwire — COMMENTED** [review]
Great stuff.

File: src/components/URLInput/URLInput.js (hunk 1 above)
Comment (jsjohnstone) [reply src/components/URLInput/URLInput.js:64]: Thanks! I spotted it just as I uploaded it - have committed the fix (removed the unused react components). Should be good to go if you're happy? :)

Comment (abirdonwire) [reply src/components/URLInput/URLInput.js:64]: Definitely, off we go!

**Review by abirdonwire — APPROVED** [review]

**Pull request merged**